packages/frontend/src/utils/staking.js

```diff
@@ -95,3 +98,33 @@ export function shuffle(sourceArray) {
     }
     return sourceArray;
 }
+
+const SECONDS_IN_YEAR = 3600 * 24 * 365;
+
+export const calculateAPY = async (validator, tokenPrices) => {
```

Comment:
For now, I can't guarantee that this will always work as expected with the regular numbers. If you think so, we can omit library usage in this case. 

Also, is there a reason, why you prefer `bn.js` rather than `big.js`?